recipes/libqasm/all/conanfile.py

```diff
@@ -0,0 +1,134 @@
+        if self._should_build_test:
+            self.test_requires("gtest/1.14.0")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(f"{self.ref} requires C++{self._min_cppstd},"
+                                            f"which your compiler does not support.")
+
+        if self.settings.arch != "wasm" and self.dependencies["antlr4-cppruntime"].options.shared != self.options.shared:
+            raise ConanInvalidConfiguration(f"{self.ref} requires antlr4-cppruntime to be built with the same shared option value.")
+
+    def requirements(self):
+        self.requires("fmt/10.2.1", transitive_headers=True)
+        self.requires("range-v3/0.12.0", transitive_headers=True)
+        self.requires("tree-gen/1.0.7", transitive_headers=True, transitive_libs=True)
+        if not self.settings.arch == "wasm":
+            self.requires("antlr4-cppruntime/4.13.1", transitive_headers=True)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        deps = CMakeDeps(self)
+        deps.generate()
+        tc = CMakeToolchain(self)
+        tc.variables["LIBQASM_BUILD_PYTHON"] = self.options.build_python
+        tc.variables["LIBQASM_BUILD_TESTS"] = self._should_build_test
+        tc.generate()
+        env = VirtualBuildEnv(self)
+        env.generate()
+
+    def _patch_sources(self):
+        if is_msvc(self):
+            replace_in_file(self, os.path.join(self.source_folder, "src", "CMakeLists.txt"), "/WX", "")
+            replace_in_file(self, os.path.join(self.source_folder, "test", "CMakeLists.txt"), "/WX", "")
+        else:
+            replace_in_file(self, os.path.join(self.source_folder, "src", "CMakeLists.txt"), "-Werror", "")
+            replace_in_file(self, os.path.join(self.source_folder, "test", "CMakeLists.txt"), "-Werror", "")
+
+    def build(self):
+        self._patch_sources()
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, "LICENSE.md", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        cmake = CMake(self)
+        cmake.install()
+        fix_apple_shared_install_name(self)
+        rm(self, "*.interp", os.path.join(self.package_folder, "include"), recursive=True)
+        rm(self, "*.pdb", self.package_folder, recursive=True)
+        rm(self, "*.tokens", os.path.join(self.package_folder, "include"), recursive=True)
+        rm(self, "cmake_install.cmake", os.path.join(self.package_folder, "include"), recursive=True)
+        rm(self, "Makefile", os.path.join(self.package_folder, "include"), recursive=True)
```

Comment:
```suggestion
        rm(self, "*.pdb", self.package_folder, recursive=True)
```
These should no longer be necessary if the CMake install() commands are working correctly.

Reply:
On it! Thanks!